docs/markdown/clusters/running_jobs.md

```diff
@@ -44,7 +44,7 @@ For more information, visit the [Nextflow documentation](https://www.nextflow.io
 
 #### Nextflow version
 
-It is advisable to keep the Nextflow version at `>= 19.10.0` which can be achieved by updating Nextflow using:
+It is advisable to keep Nextflow at the newest version which can be achieved by updating Nextflow using:
```

Comment:
The newest version can be seen on the nextflow GitHub page, therefore the link might be useful
```suggestion
It is advisable to keep [Nextflow](https://github.com/nextflow-io/nextflow) at the newest version which can be achieved by updating Nextflow using:
```